.github/workflows/test.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       - id: fork-check
         run: |
           if [[ "${{ github.event.repository.full_name}}" == "lf-edge/eve" ]] || [[ "${{ github.event.repository.full_name}}" == "lf-edge/eden" ]]; then
-            echo "runner=buildjet-4vcpu-ubuntu-2204" >> "$GITHUB_OUTPUT"
+            echo "runner=[buildjet-4vcpu-ubuntu-2204, buildjet-pinned-7950x]" >> "$GITHUB_OUTPUT"
```

Comment:
Would it be possible to apply this only for Virtualization test suite?
Otherwise we may limit ourselves to only few servers and have many workflows stalled in the Queued state.

Reply:
Maybe it could output 'runner' and 'runner-virt' as two variables and use 'runner-virt' [here](https://github.com/lf-edge/eden/blob/master/.github/workflows/test.yml#L188)?